person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(id, age, name = 'Unknown')
```

Comment:
- Since we are retrieving the id once, we can remove it from the constructor and update the constructor like so below:
`def initialize(age, name = "Unknown", parent_permission = true)`